os/arch/arm/src/armv7-r/arm_initialstate.c

```diff
@@ -112,6 +112,10 @@ void up_initial_state(struct tcb_s *tcb)
 
 	xcp->regs[REG_PC] = (uint32_t)tcb->start;
 
+#if defined(CONFIG_BUILD_PROTECTED)
+	up_setup_regions(tcb, tcb->flags);
```

Comment:
Why to add protected build changes again?

Reply:
@satheesh-ks please check this

Reply:
Hi All,
This patch is simple revert of commit 2dbd158de140bf14ec4ac471ee9640542708f8bc.
So in the original patch it might have in "-" section which might be added as "+" section in my reversion